Nativing/accounts/templates/accounts_signup.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+
+    <form method="POST" action="{% url 'accounts_signup' %}">
```

Comment:
```suggestion
    <form method="POST" action="{% url 'accounts_signup' %}" enctype="multipart/form-data">
```
이미지 파일을 업로드 하려면 필요합니다!